cmd/minishift/cmd/start_preflight.go

```diff
@@ -37,23 +36,25 @@ import (
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/viper"
 
-	cmdUtils "github.com/minishift/minishift/cmd/minishift/cmd/util"
+	cmdUtil "github.com/minishift/minishift/cmd/minishift/cmd/util"
 	minishiftNetwork "github.com/minishift/minishift/pkg/minishift/network"
 	openshiftVersion "github.com/minishift/minishift/pkg/minishift/openshift/version"
 	stringUtils "github.com/minishift/minishift/pkg/util/strings"
 )
 
 const (
 	StorageDisk   = "/mnt/?da1"
-	GithubAddress = "github.com:http"
+	GithubAddress = "https://github.com"
 )
 
 // preflightChecksBeforeStartingHost is executed before the startHost function.
 func preflightChecksBeforeStartingHost() {
 	driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic (https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html) for more information"
 	prerequisiteErrorMessage := "See the 'Installing Prerequisites for Minishift' topic (https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites) for more information"
 
+	fmt.Printf("-- Checking if %s is reachable (using proxy: %q) ... ", GithubAddress, os.Getenv("HTTPS_PROXY"))
```

Comment:
Doesn't this printout every time no matter if the proxy is set or not? We can put this behind an condition where we first check if there is PROXY env set or not.

Reply:
Yes, it does but I think that it's good to be explicitly made aware of the fact that no proxy is in use at that point. Now, of course, this is a somewhat selfish use case because I use proxies and I consider it likewise helpful to see which proxy I am using and whether I'm using any proxy at all.

I vote for keeping some time of indication around in this line that will tell you that you are not using a proxy. I think considering that this meant to be an Internet connectivity check, we can justify keeping a proxy message around.

Reply:
@svenstaro I am all fine with having it part of explicit but then also I would not print out the `HTTPS_PROXY` variable directly to stdout because most of the time it contains the proxy info along with credential and during the demo or presenting to someone might expose this :(

Reply:
Good point, will fix.